src/Polymath.ts

```diff
@@ -431,6 +435,32 @@ export class Polymath {
     return await procedure.prepare();
   };
 
+  public forceTransfer = async (args: {
+    securityTokenId: string;
+    value: BigNumber;
+    from: string;
+    to: string;
+    log: string;
+    data: string;
```

Comment:
This `data` parameter is supposedly meant for validation of a signature, but it's not currently being used for anything and is sent as an empty string. For now, it should be optional as well and default to an empty string